public/data/bad-belzig.geojson

```diff
@@ -1 +1 @@
"coordinates":[[[12.5246936,52.1922112],[12.5248267,52.1952747],[12.5314142,52.1959192],[12.5402976,52.198497],[12.541731,52.1987962],[12.5436997,52.1987198],[12.5484633,52.1974178],[12.5498366,52.1965366],[12.5517034,52.1935771],[12.5562739,52.1905912],[12.5591707,52.1847898],[12.5629312,52.1819184],[12.5684833,52.1814743],[12.5741911,52.1808428],[12.5757146,52.1808954],[12.5783539,52.1801849],[12.5836754,52.1806191],[12.5875592,52.1804349],[12.5878811,52.1798428],[12.5912285,52.1801191],[12.5946402,52.1808165],[12.5995541,52.1820532],[12.6021719,52.1824479],[12.6051545,52.1825137],[12.6056695,52.1817111],[12.6072144,52.1809743],[12.6079869,52.1752243],[12.6052403,52.1732109],[12.6045537,52.1716318],[12.6076651,52.1711448],[12.6086736,52.1704737],[12.6086092,52.1686312],[12.6109266,52.1651961],[12.6129866,52.165354],[12.6136464,52.1666465],[12.6149875,52.1661158],[12.6134801,52.1637351],[12.6117206,52.1630638],[12.6090169,52.1606813],[12.6089525,52.1601021],[12.6102614,52.1567849],[12.6117045,52.1561503],[12.610712,52.1553894],[12.6086521,52.1553631],[12.6069784,52.1539808],[12.6170195,52.1504102],[12.6161021,52.1451691],[12.6161236,52.1436442],[12.6166815,52.1426698],[12.618441,52.1413002],[12.6165527,52.1402862],[12.6153296,52.1350972],[12.6176042,52.134465],[12.6168531,52.1295652],[12.6146215,52.1294993],[12.6153511,52.1257714],[12.6131624,52.1250205],[12.6066393,52.1243486],[12.605266,52.1234923],[12.602455,52.1233869],[12.6008886,52.1229653],[12.5998801,52.1234264],[12.5985712,52.1229258],[12.5966185,52.1230575],[12.5977129,52.1198297],[12.5922626,52.1191446],[12.5877576,52.119221],[12.5874347,52.1167861],[12.5902671,52.1166807],[12.5940007,52.1156002],[12.5975841,52.1149809],[12.5977987,52.1138609],[12.5916833,52.1135447],[12.5914912,52.1111701],[12.5937665,52.1100607],[12.5909521,52.1082873],[12.5871489,52.1080688],[12.5815248,52.1084786],[12.5766201,52.1092068],[12.5701881,52.1106452],[12.5644804,52.1121343],[12.5621844,52.1123451],[12.5593295,52.1118852],[12.5601674,52.1158766],[12.5609184,52.1181244],[12.5610794,52.1197582],[12.5602318,52.1227475],[12.5596095,52.1239806],[12.5615085,52.1263769],[12.5583006,52.1270224],[12.553537,52.1285505],[12.55233,52.1294047],[12.5505705,52.1320918],[12.5509191,52.1332529],[12.5498892,52.1369802],[12.5514556,52.1377177],[12.5512196,52.1385474],[12.54946,52.1404307],[12.5472928,52.1432883],[12.5469066,52.145461],[12.5472928,52.1464617],[12.5432373,52.1474361],[12.541113,52.147739],[12.5407697,52.1492155],[12.5414134,52.1509666],[12.5439454,52.1543107],[12.5452544,52.158181],[12.5457264,52.1582205],[12.5480439,52.1634463],[12.5442673,52.1641176],[12.5448252,52.1679344],[12.5483614,52.1692077],[12.5507475,52.1710403],[12.5513483,52.1740934],[12.5522453,52.1768666],[12.5461084,52.1809587],[12.5421859,52.1830671],[12.5395037,52.183317],[12.5395209,52.1849846],[12.5341779,52.1864975],[12.529336,52.1892232],[12.5261592,52.190763],[12.5246936,52.1922112]]]},"properties":{"id":-540424,"name":"Bad Belzig"}}]}
```

Comment:
Could you please add a newline at the very end of the file? :)

Reply:
sure, done